extensions/sql-migration/src/dashboard/dashboardTab.ts

```diff
@@ -764,11 +765,10 @@ export class DashboardTab extends TabBase<DashboardTab> {
 			})
 			.component();
 
-		const connectionProfile = await azdata.connection.getCurrentConnection();
 		this.disposables.push(
 			this.serviceContextChangedEvent.event(
 				async (e) => {
-					if (e.connectionId === connectionProfile.connectionId) {
+					if (e.connectionId === await getSourceConnectionId()) {
```

Comment:
due to the implementation of this event, the code gets executed in all open instances of the extension... i.e. one per server connection.
 This logic was added to ensure that the Connection ID sending the event was the same one receiving it...  To test open up multiple instances/connections to two or more servers, then set different migration service contexts...  then ensure it doesn't change for the other open sql-migration dashboards related to/opened on different server connections